In case of hardware forwarding with aging in kernel, update fdb "used" on receiving rtnl message for hit bit update in the hardware.

diff --git a/drivers/net/vxlan.c b/drivers/net/vxlan.c
index b89901c..85dfad8 100644
--- a/drivers/net/vxlan.c
+++ b/drivers/net/vxlan.c
@@ -283,6 +283,14 @@ static struct vxlan_fdb *vxlan_find_mac(struct vxlan_dev *vxlan,
 static int vxlan_fdb_replace(struct vxlan_fdb *f, __be32 ip)
 {
 	f->remote_ip = ip;
+	/* In case of hardware forwarding with aging in kernel, update
+	 * fdb when the corresponding entry in hardware gets used. As
+	 * there is no flag to identify hardware or software forwarding,
+	 * this has the side effect of updating "used" in software forwarding
+	 * as well if there is any change in the entry. Also, update happens
+	 * for mac move from one remote node to another.
+	 */
+	f->used = jiffies;
 	return 1;
 }
 
